clang/lib/CConv/AVarBoundsInfo.cpp

```diff
@@ -674,9 +674,9 @@ bool AVarBoundsInfo::performFlowAnalysis(ProgramInfo *PI) {
       unsigned ParmNum = std::get<3>(ParmTup);
       FVConstraint *FV = nullptr;
       if (IsStatic || !PI->getExtFuncDefnConstraintSet(FuncName)) {
-        FV = getOnly(*(PI->getStaticFuncConstraintSet(FuncName, FileName)));
+        FV = PI->getStaticFuncConstraintSet(FuncName, FileName);
```

Comment:
Love it! 

But: Now the name `getStaticFuncConstraintSet` needs to change, since it's not returning a set, right?

Reply:
Yeah, let me know about other such name changes that should happen. I caught a few, but I'm sure they're more.

Reply:
Will do (added comments nearby the things whose names shoudl change)